Feature_Guide/Speculative_Decoding/vLLM/model_repository/eagle_model copy/config.pbtxt

```diff
@@ -0,0 +1,37 @@
+# Copyright 2025, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Question - I didn't see this "eagle_model copy " used in the tutorial, and was wondering why we'd need this?